src/mumble/Themes.cpp

```diff
@@ -78,9 +88,21 @@ bool Themes::applyConfigured() {
 	skinPaths << qssFile.path();
 	skinPaths << QLatin1String(":/"); // Some skins might want to fall-back on our built-in resources
 	QDir::setSearchPaths(QLatin1String("skin"), skinPaths);
+
+	QString themeQss = QString::fromUtf8(file.readAll());
+
+	QString userStylesheetFn = userStylesheetPath();
+	QString userStylesheetContent;
+	if (readStylesheet(userStylesheetFn, userStylesheetContent)) {
+		qWarning("Themes: allowing user stylesheet at '%s' to override the chosen theme", qPrintable(userStylesheetFn));
+	}
 	
-	qApp->setStyleSheet(QString::fromUtf8(file.readAll()));
-	
+	qApp->setStyleSheet(
+		themeQss +
+		QLatin1String("\n") +
+		userStylesheetContent
+	);
+
```

Comment:
This code is almost identical to what's above.

Have you considered moving both of them into a single function?  It might help maintainability and readability.